kuma/wiki/templates/wiki/email/edited.ltxt

```diff
@@ -32,4 +32,8 @@ Edit Article:
 Article History:
 {% endtrans %}
  {{ history_url|absolutify }}
-{% if watch %}{{ unsubscribe_text(watch) }}{% endif %}{% endautoescape %}
+--
+{% if watch %}
+{% trans %}You are subscribed to edits on:{% endtrans %} {{ watch.content_object.title }}{% if watch.event_type == 'wiki edit document in tree' %}{% trans %} and all its sub-articles{% endtrans %}{% endif %}.
```

Comment:
I don't think it is good to break apart a sentence into multiple chunks like this for l10n. I imagine RTL languages would have a difficult time with this.

Would something like this be easier for localizers?
```
{% if watch %}
  {% set title = watch.content_object.title %}
  {% if watch.event_type == 'wiki edit document in tree' %}
    {{ _('You are subscribed to edits on: {title} and all its sub-articles.')|f(title=title) }}
  {% else %}
    {{ _('You are subscribed to edits on: {title}.')|f(title=title) }}
  {% endif %}
{% endif %}
```